File: PubeoAPI/Model/Participation.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace PubeoAPI.model {
    public class Participation {
        
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        [Required]
        public Guid ParticipationId { get; set; }
        public Guid ParticulierId { get; set; }

        [ForeignKey("ParticulierId")]
        public virtual Particulier Particulier { get; set; }
        [Required]
        public Guid StickerId { get; set; }

        [ForeignKey("StickerId")]
        public virtual Sticker Sticker { get; set; }
    }
}

File: PubeoApp/app/src/main/java/com/example/pubeo/model/Participation.java
package com.example.pubeo.model;

public class Participation {

    private String participationId;
    private String particulierId;
    private String stickerId;
    private Particular particulier;
    private Sticker sticker;

    public Participation(String participationId, String particulierId, String stickerId, Particular particulier, Sticker sticker) {
        this.participationId = participationId;
        this.particulierId = particulierId;
        this.stickerId = stickerId;
        this.particulier = particulier;
        this.sticker = sticker;
    }

    public String getParticipationId() {
        return participationId;
    }

    public String getParticulierId() {
        return particulierId;
    }

    public String getStickerId() {
        return stickerId;
    }

    public Particular getParticulier() {
        return particulier;
    }

    public Sticker getSticker() {
        return sticker;
    }
}
